### PR TITLE
fix: fixed a bug that fitted props did not work

### PR DESCRIPTION
**Pull request by YSL0618**
Closes # https://github.com/yamada-ui/yamada-ui/issues/4184

## Description
Fixed a bug that `fitted` props did not work in `Tabs`.

### CODE REVIEW

File: .changeset/ninety-vans-develop.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@yamada-ui/tabs": minor
```
Comment (hirotomoyamada) [inline .changeset/ninety-vans-develop.md:2]: Please change minor to patch.

File: .changeset/ninety-vans-develop.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+"@yamada-ui/theme": minor
```
Comment (hirotomoyamada) [inline .changeset/ninety-vans-develop.md:3]: Please change minor to patch.

Comment (YSL0618) [reply .changeset/ninety-vans-develop.md]: I have fixed the changeset file.

**Review by hirotomoyamada — APPROVED** [review]
Thanks!

**Pull request merged**